pyglidein/etc/client_defaults.cfg

```diff
@@ -22,5 +22,9 @@ max_gpus_per_job = 8
 max_memory_per_job = 24000
 max_disk_per_job = 100000
 
+# command settings
+running_cmd = condor_q -all -constraint 'Owner == "${USER}" && JobStatus == 2' -af ClusterID | wc -l
+idle_cmd = condor_q -constraint 'Owner == "${USER}" && JobStatus == 1' -af ClusterId | wc -l
+
```

Comment:
I'm hesitant to put condor commands as the global default.  If the condor command is not found, what happens?  Does this throw an error and stop execution?

Reply:
Good call.  I think I would like to start generating Scheduler Specific Default Configs in the near future, but this isn't the PR for that.  I'll move these back to my local config.